ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-extended-community-lists-regexp.j2

```diff
@@ -1,9 +1,9 @@
 {# eos - ip-extended-community-lists-regexp #}
-{% for ip_extcommunity_list in ip_extcommunity_lists_regexp | arista.avd.natural_sort %}
+{% for ip_extcommunity_list in ip_extcommunity_lists_regexp | arista.avd.convert_dicts('name','types') | arista.avd.natural_sort('name') %}
 !
-{%     for sequence in ip_extcommunity_lists_regexp[ip_extcommunity_list] | arista.avd.default([]) %}
+{%     for sequence in ip_extcommunity_list.types | arista.avd.default([]) %}
```

Comment:
please change the item var to `entry` so `for entry in ip_extcommunity_list.entries` to make it more readable